go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/boltdb/bolt
```

Comment:
Perhaps this should be `module github.com/Psiphon-Labs/bolt`?

Reply:
Yes. Thanks. Fix: https://github.com/Psiphon-Labs/bolt/commit/f3d58e369264bf53342d4ac95aec3355901c957e.